modules/components/pages/redpanda/about.adoc

```diff
@@ -4,9 +4,12 @@ component_type_dropdown::[]
 
 // tag::single-source[]
 
-// © 2024 Redpanda Data Inc.
+The Redpanda Connect configuration service allows you to:
```

Comment:
Could we change the nav item to call this the Redpanda Connect configuration service or something similar?